BraveWallet/Crypto/CryptoPagesView.swift

```diff
@@ -141,6 +149,19 @@ private class CryptoPagesViewController: TabbedPageViewController {
     }
     
     swapButton.addTarget(self, action: #selector(tappedSwapButton), for: .touchUpInside)
+    
+//    let confirmationsButton = SwapButton()
+//    view.addSubview(confirmationsButton)
+//    confirmationsButton.snp.makeConstraints {
+//      $0.trailing.equalToSuperview().inset(16)
+//      $0.bottom.equalTo(view.safeAreaLayoutGuide).priority(.high)
+//      $0.bottom.lessThanOrEqualTo(view).inset(8)
+//    }
+//    confirmationsButton.addTarget(self, action: #selector(tappedConfirmationsButton), for: .touchUpInside)
+  }
```

Comment:
comments? or remove it?

Reply:
Just left it as a reference point for where a button is going to go for viewing pending confirmations that the user didn't complete, will replace it in during integration